src/network/netdev/fou-tunnel.c

```diff
@@ -112,6 +113,36 @@ static int netdev_fou_tunnel_create(NetDev *netdev) {
         return 0;
 }
 
+int config_parse_ip_protocol(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        uint8_t *protocol = data;
+        int r;
+
+        assert(filename);
+        assert(section);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        r = parse_ip_protocol(rvalue);
+        if (r < 0 || r > 255)
```

Comment:
this is not going to return > 255, is it?

Reply:
Yeah. I changed the redundant check to `assert_cc()`.